### DISCUSSION TITLE
Profile view

### DISCUSSION THREAD

**Discussion by renja-g** [anuraghazra/github-readme-stats#2600] [Q&A] [answered]
Hi, I have a question, what do I need to do so that my profile is public?

The dashboard works:
![image](https://user-images.githubusercontent.com/76645494/230079880-9f2d42df-35bb-4c33-b652-dfd0058da130.png)

But the profile does not:
![image](https://user-images.githubusercontent.com/76645494/230080083-72427cf2-3065-45ee-9143-0643ec61dc5a.png)

Thea's are my current profile settings:
![image](https://user-images.githubusercontent.com/76645494/230080299-c056c3f3-18a4-448c-b69a-41dcaec994a3.png)

I found this but I think I have the settings right.
#1092 

I also found this one, but `&v=2` does not help, ofc. I also tried clearing the cash.
#515

**Comment by rickstaa** [comment, marked as answer]
I guess that at the time of writing, your wakatime profile was still under review. I checked it today and now it is public:

```md
[![willianrod's wakatime stats](https://github-readme-stats.vercel.app/api/wakatime?username=renja)](https://github.com/anuraghazra/github-readme-stats)
````

[![willianrod's wakatime stats](https://github-readme-stats.vercel.app/api/wakatime?username=renja)](https://github.com/anuraghazra/github-readme-stats)
